Sources/SwiftInspectorAnalyzers/PropertyAnalyzer.swift

```diff
@@ -21,292 +21,18 @@
 // SOFTWARE.
```

Comment:
Putting this here so we have a thread. Can I get access to CloudCov?

![Screen Shot 2021-04-08 at 1 04 07 PM](https://user-images.githubusercontent.com/1791049/114089428-fb991100-986a-11eb-9002-778918a3ba14.png)

Reply:
You should have access https://app.codecov.io/gh/fdiaz/SwiftInspector

The links on the report don't work for me either 🤔

Reply:
That did work, though this should def be easier. Thanks Dan.